query/src/interpreters/interpreter_factory.rs

```diff
@@ -124,9 +124,7 @@ impl InterpreterFactory {
             PlanNode::Show(ShowPlan::ShowRoles(v)) => {
                 ShowRolesInterpreter::try_create(ctx_clone, v)
             }
-            PlanNode::Show(ShowPlan::ShowStages(v)) => {
-                ShowStagesInterpreter::try_create(ctx_clone, v)
-            }
+            PlanNode::Show(ShowPlan::ShowStages) => ShowStagesInterpreter::try_create(ctx_clone),
```

Comment:
```suggestion
            PlanNode::Show(_) => ShowStagesInterpreter::try_create(ctx_clone),
```

Reply:
It can't be ignored.

Since 

```

pub enum ShowPlan {
    ShowDatabases(ShowDatabasesPlan),
    ShowTables(ShowTablesPlan),
    ShowEngines(ShowEnginesPlan),
    ShowFunctions(ShowFunctionsPlan),
    ShowMetrics(ShowMetricsPlan),
    ShowProcessList(ShowProcessListsPlan),
    ShowSettings(ShowSettingsPlan),
    ShowUsers(ShowUsersPlan),
    ShowGrants(ShowGrantsPlan),
    ShowRoles(ShowRolesPlan),
    ShowTabStat(ShowTabStatPlan),
    ShowStages,
}
```

Reply:
> It can't be ignored.
> 
> Since
> 
> ```
> 
> pub enum ShowPlan {
>     ShowDatabases(ShowDatabasesPlan),
>     ShowTables(ShowTablesPlan),
>     ShowEngines(ShowEnginesPlan),
>     ShowFunctions(ShowFunctionsPlan),
>     ShowMetrics(ShowMetricsPlan),
>     ShowProcessList(ShowProcessListsPlan),
>     ShowSettings(ShowSettingsPlan),
>     ShowUsers(ShowUsersPlan),
>     ShowGrants(ShowGrantsPlan),
>     ShowRoles(ShowRolesPlan),
>     ShowTabStat(ShowTabStatPlan),
>     ShowStages,
> }
> ```

Oh, I see. Never mind.